### PR TITLE
US21-Task74 Add from and to date option on the UI

### PR DESCRIPTION
**Pull request by spark290**
US21 Task74 Add from and to date option on the UI
Screenshot

### CODE REVIEW

File: react-ui/src/date/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                            <Form.Control type="date" value={startDate} onChange={e => setStartDate(e.target.value)} />
+                        </Form.Group>
+                        <Form.Group controlId="dateTo" style={{ width: '15%' }}>
+                            <Form.Label>To</Form.Label>
+                            <Form.Control type="date" value={endDate} onChange={e => setEndDate(e.target.value)} />
+                        </Form.Group>
+                    </div>
+                </div>
+                <div style={{ display: 'flex', justifyContent: 'center' }}>
+                    <Button variant="primary" type="submit" className="submitButton backgroundButton">Select Date</Button>
```
Comment (shahih25) [inline react-ui/src/date/index.js]: Can you rename the button to **Submit**?

**Comment by shahih25** [comment]
Why do you have 2 folders **date** and **devfocus**?

File: react-ui/src/date/index.js (hunk 1 above)
Comment (spark290) [reply react-ui/src/date/index.js]: screenshot
Done

**Review by shahih25 — APPROVED** [review]

**Comment by AdvaitChirmule** [comment]
Please reflect the recent styling changes

**Comment by spark290** [comment]
> Please reflect the recent styling changes

Screenshots

**Review by AdvaitChirmule — APPROVED** [review]
Looks good to me

**Pull request merged**